### PR TITLE
Custom tags + Docker Support

### PR DESCRIPTION
**Pull request by chaos-baum**
Refactored the code a little bit to link to human readable sites when accessing the link attribute. Also improved the date accesing a little bit. 

Addtionally I introduced a new namespace so the rss feed includes all data that is returned by the API.

### CODE REVIEW

File: lfm/templates/loved.html [hunk 1]
```diff
@@ -1,13 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<rss version="2.0" xmlns:dc="http://purl.org/dc/elements/1.1/">
+<rss version="2.0" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:lfm="https://www.last.fm/api">
 	<channel>
 		<title>{{context.user}} - loved tracks</title>
         <lastBuildDate>{{ ''|rfc822_date() }}</lastBuildDate>
         <language>en</language>
-		<link>https://lfm.xiffy.nl/{{context.user}}/{{context.type}}</link>
+        <link>https://www.last.fm/user/{{context.user}}/loved</link>
```
Comment (xiffy) [inline lfm/templates/loved.html:7]: I disagree with this line. The <link> tag inside RSS should point to self, so that means the original line of code.

**Review by xiffy — COMMENTED** [review]
Beside that one comment, looks good to me and a nice addition for those who read extra namespaces ;-)

File: lfm/templates/loved.html (hunk 1 above)
Comment (xiffy) [reply lfm/templates/loved.html:7]: i stand corrected from the rss-spec:
The URL to the HTML website corresponding to the channel.

**Comment by xiffy** [comment]
it's live ;-)
Cheers

**Comment by chaos-baum** [comment]
Thanks for the quick check ;)

**Pull request merged**